utils.py

```diff
@@ -7,9 +7,9 @@ def setup_logging(path='config/logging.yaml',
                   level=logging.INFO,
                   env_key='LOG_CFG'):
 
-    value = os.getenv(env_key, path)
-    if value and os.path.exists(value):
-        with open(path, 'rt') as f:
+    log_file_path = os.getenv(env_key, path)
+    if log_file_path and os.path.exists(log_file_path):
```

Comment:
I have set up a default `config/logging.yaml in such a way that it get logged to both stdout and log file.